engine/execution/ingestion/engine.go

```diff
@@ -81,9 +78,6 @@ func New(
 	headers storage.Headers,
 	blocks storage.Blocks,
 	collections storage.Collections,
-	events storage.Events,
-	serviceEvents storage.ServiceEvents,
-	transactionResults storage.TransactionResults,
```

Comment:
These unused dependencies are needed by GRPC engine, but not the ingestion engine